File: app_java/src/main/java/com/example/mvvmdemo/view/adapter/BindingAdapters.java
package com.example.mvvmdemo.view.adapter;

import android.view.View;

import androidx.databinding.BindingAdapter;

public class BindingAdapters {

    @BindingAdapter("visibleGone")
    public static void isShow(View view, boolean show) {
        view.setVisibility(show ? View.VISIBLE : View.GONE);
    }



}

File: app_kotlin/src/main/java/com/example/app_kotlin/view/adapter/BindingAdapters.kt
package com.example.app_kotlin.view.adapter

import android.view.View
import androidx.databinding.BindingAdapter

class BindingAdapters {

    companion object {
        @JvmStatic
        @BindingAdapter("visibleGone")
        fun isShow(view: View, show: Boolean) {
            view.visibility = if (show) View.VISIBLE else View.GONE
        }
    }
}